user_service/src/controllers/UserController.ts

```diff
@@ -43,4 +44,54 @@ export const delUserProfile = async (req: any, res: any) => {
             error: error,
             message: 'User not found.'})
     }
+}
+
+
+//@desc     get user's answered questions
+//@route    GET /api/users/:id/questions
+//@access   authenticated users
+export async function getUserQuestions(req: any, res: any) {
+  try {
+    const userId = req.params.userId;
```

Comment:
`req.params.userId` will always return null because `userID` is not a param of the route. 

should be `req.params.id`